docs/IE_PLUGIN_DG/LowPrecisionModelRepresentation.rst

```diff
@@ -1,21 +0,0 @@
-.. {#openvino_docs_ie_plugin_dg_lp_representation}
-
-Representation of low-precision models
-======================================
-The goal of this document is to describe how optimized models are represented in OpenVINO Intermediate Representation (IR) and provide guidance on interpretation rules for such models at runtime. 
-Currently, there are two groups of optimization methods that can influence on the IR after applying them to the full-precision model:
-- **Sparsity**. It is represented by zeros inside the weights and this is up to the hardware plugin how to interpret these zeros (use weights as is or apply special compression algorithms and sparse arithmetic). No additional mask is provided with the model.
-- **Quantization**. The rest of this document is dedicated to the representation of quantized models.
```

Comment:
why is this file removed?
It's a part of the docs https://docs.openvino.ai/2023.2/openvino_docs_ie_plugin_dg_lp_representation.html and referred from here 
![image](https://github.com/openvinotoolkit/openvino/assets/2566854/11abb886-b354-4087-a78e-7e5fac6113b0)

Reply:
The file has been moved to the new location: 
docs/articles_en/documentation/openvino_extensibility/openvino_plugin_library/detailed_guides/QuantizedNetworks/LowPrecisionModelRepresentation.rst

Reply:
thanks, missed that